app/controllers/application_controller.rb

```diff
@@ -1,2 +1,29 @@
+require 'net/http'
 class ApplicationController < ActionController::API
```

Comment:
Now that we have more than one controller, I'm not sure how comfortable I am with all of this logic going into application controller which is supposed to be a shared super class for all controllers.  The expectation in rails is that unless otherwise specified, you would inherit from `ApplicationController`.

What about `RMAPIProxyController` since that's effectively what happens.